solo/docker/start.sh

```diff
@@ -23,7 +23,8 @@ done
 curl -XPUT http://127.0.0.1:4001/v2/keys/skydns/${SKYDNS_PATH} \
     -d value="{\"host\":\"$HOST_ADDRESS\"}"
 
-skydns $SKYDNS_OPTS &
+skydns $SKYDNS_OPTS -verbose &
```

Comment:
I'm not sure if `-verbose` should be put in the image. Seems like you could run the docker image with `-e SKYDNS_OPTS=-verbose` at runtime if you really wanted verbose logging.